### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by priyatharsan**
closes: https://github.com/Aifred-Health/Vulcan/issues/111, https://github.com/Aifred-Health/Vulcan/issues/112

### CODE REVIEW

File: vulcanai/models/metrics.py [hunk 1]
```diff
@@ -61,6 +61,9 @@ def get_score(targets, predictions, metrics='accuracy', average=None,
         """
         double_parametered_functions = ["get_accuracy"]
 
+        if isinstance(targets, torch.Tensor):
+            targets = targets.cpu().numpy()
```
Comment (rfratila) [inline vulcanai/models/metrics.py]: this should be `targets = targets.cpu().detach().numpy()`

File: vulcanai/plotters/visualization.py [hunk 2]
```diff
@@ -401,7 +401,7 @@ def display_receptive_fields(network, top_k=5, save_path=None):
     fig.suptitle("Feature importance")
     num_layers = len(network._modules['network'])
     for i, layer in enumerate(network._modules['network']):
-        raw_field = layer._kernel._parameters['weight'].detach().numpy()
+        raw_field = layer._kernel._parameters['weight'].detach().cpu().numpy()
```
Comment (rfratila) [inline vulcanai/plotters/visualization.py]: could you put the `.cpu()` before the `.detach()`?

File: vulcanai/models/metrics.py (hunk 1 above)
Comment (priyatharsan) [reply vulcanai/models/metrics.py]: 👍

File: vulcanai/plotters/visualization.py (hunk 2 above)
Comment (priyatharsan) [reply vulcanai/plotters/visualization.py]: 👍
